sdks/java/io/jms/src/main/java/org/apache/beam/sdk/io/jms/JmsIO.java

```diff
@@ -123,11 +131,13 @@
 public class JmsIO {
 
   private static final Logger LOG = LoggerFactory.getLogger(JmsIO.class);
+  private static final long DEFAULT_CLOSE_TIMEOUT = 1000L;
```

Comment:
Is one second appropriate?

Reply:
Indeed no, the tests were good with 60s. We may be able to support a lower value, but not for sure.